File: samples-vb/examples/BinaryClasification_Titanic/BinaryClasification_TitanicSurvivalPrediction/TitanicData.vb
Imports Microsoft.ML.Runtime.Api

Public Class TitanicData
    <Column("0")>
    Public PassengerId As Single
    <Column("1", "Label")>
    Public Survived As Single
    <Column("2")>
    Public Pclass As Single
    <Column("3")>
    Public Name As String
    <Column("4")>
    Public Sex As String
    <Column("5")>
    Public Age As Single
    <Column("6")>
    Public SibSp As Single
    <Column("7")>
    Public Parch As Single
    <Column("8")>
    Public Ticket As String
    <Column("9")>
    Public Fare As String
    <Column("10")>
    Public Cabin As String
    <Column("11")>
    Public Embarked As String
End Class

File: samples/csharp/getting-started/BinaryClasification_Titanic/TitanicSurvival/TitanicSurvivalConsoleApp/DataStructures/TitanicData.cs
using Microsoft.ML.Runtime.Api;

namespace TitanicSurvivalConsoleApp.DataStructures
{
    public class TitanicData
    {
        [Column("0")]
        public float PassengerId;

        [Column(ordinal: "1")]
        public float Label;   //If "Survived" or not

        [Column("2")]
        public float Pclass;

        [Column("3")]
        public string Name;

        [Column("4")]
        public string Sex;

        [Column("5")]
        public float Age;

        [Column("6")]
        public float SibSp;

        [Column("7")]
        public float Parch;

        [Column("8")]
        public string Ticket;

        [Column("9")]
        public float Fare;

        [Column("10")]
        public string Cabin;

        [Column("11")]
        public string Embarked;
    }
}